### ISSUE TITLE
Change orientation

### ISSUE THREAD

**Issue by tjakovcevic** [mochixuan/react-native-drag-sort#54]
I love this package, saved my project!
I have an issue though when changing orientation on Android.
I cannot get it to work. 

This is what I use to detect orientation change (this works):
```
Dimensions.addEventListener("change", function (dims) {
            self.setState({
                deviceWidth: dims.screen.width,
                deviceHeight: dims.screen.height
            });
        });
```

This is what I use on the package:

```
let delimiter = (this.state.deviceWidth > this.state.deviceHeight) ? 7 : 4;
<AutoDragSortableView
                        childrenWidth={this.state.deviceWidth / delimiter -  PixelRatio.getPixelSizeForLayoutSize(1.3)}
                        childrenHeight={80}
                        sortable={!this.state.deleteMode}
                        marginChildrenTop={PixelRatio.getPixelSizeForLayoutSize(1)}
                        marginChildrenLeft={PixelRatio.getPixelSizeForLayoutSize(1)}
                        parentWidth={this.state.deviceWidth}
                        dataSource={this.props.photoData[this.state.selectedObject]}
                        onDataChange={(data) => this._orderChange(data)}
                        keyExtractor={(item, index) => item.id}
                        onClickItem={(data, item, index) => { this._openGallery(index) }}
                        renderItem={this._renderListItem} />
```

If I open the screen vertically or horizontally it works, but if I change orientation while in screen it messes up. I tried to change a few things but I got no progress, is this not implemented or am I doing something wrong?
Thank you!

**Comment by mochixuan** [comment]
Thanks for using it, I will fix it immediately, it is expected to take 1 hour.

**Comment by mochixuan** [comment]
This issue has been fixed, you can check version 2.0.3

**Comment by tjakovcevic** [comment]
Amazing, works great! Thank you for your fast response!
